org.kde.kstars.json

```diff
@@ -65,28 +72,107 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-4.6.2.tar.gz",
+                    "url": "https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-4.6.2.tar.gz",
                     "sha256": "66fd078cc0bea896b0d44b120d46d6805421a5361d3a5ad84d9f397b1b5de2cb",
-                    "disable-http-decompression": true,
                     "x-checker-data": {
                         "type": "anitya",
                         "project-id": 270,
                         "stable-only": true,
-                        "url-template": "http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-$version.tar.gz"
+                        "url-template": "https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-$version.tar.gz"
                     }
                 }
             ]
         },
         {
-            "name": "libraw",
+            "name": "wcslib",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://src.rrz.uni-hamburg.de/files/src/wcslib/wcslib-7.10.tar.bz2",
+                    "sha256": "1796b0979df950ba7eae1010b986134067187846892b8e9b3c42d30361c9d929"
+                }
+            ]
+        },
+        {
+            "name": "stellarsolver",
+            "buildsystem": "cmake-ninja",
             "builddir": true,
-            "config-opts": [
-                "--disable-examples",
-                "--disable-static"
+            "config-opts": ["-DUSE_QT5=OFF"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/rlancaste/stellarsolver/archive/refs/tags/2.7.tar.gz",
+                    "sha256": "36ff495f325920b9a23b7c3f7453a99c8d405e0984d4b27da7568625cb6c76c9",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 242699,
+                        "stable-only": true,
+                        "url-template": "https://github.com/rlancaste/stellarsolver/archive/refs/tags/$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "libev",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dist.schmorp.de/libev/Attic/libev-4.33.tar.gz",
+                    "sha256": "507eb7b8d1015fbec5b935f34ebed15bf346bed04a11ab82b8eee848c4205aea",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 1605,
+                        "stable-only": true,
+                        "url-template": "https://dist.schmorp.de/libev/Attic/libev-$version.tar.gz"
+                    }
+                }
+            ],
+            "buildsystem": "autotools"
```

Comment:
this is weird placing of this after the sources definition. It's default so you may drop it altogether